src/utilities/utility.ts

```diff
@@ -127,6 +130,81 @@ export function annotateChildPods(kubectl: Kubectl, resourceType: string, resour
     return commandExecutionResults;
 }
 
+export async function getFilePathsConfigs(): Promise<any> {
+
+    let filePathsConfig: any = {};
+    const MANIFEST_PATHS_KEY = 'manifestFilePaths';
+    const HELM_CHART_KEY = 'helmChartFilePaths';
+    const DOCKERFILE_PATH_KEY = 'dockerfilePaths';
+    const DOCKERFILE_PATH_LABEL_KEY = 'dockerfile-path';
+
+    let inputManifestFiles = inputParams.manifests || [];
+    filePathsConfig[MANIFEST_PATHS_KEY] = inputManifestFiles;
+
+    let helmChartPaths = (process.env.HELM_CHART_PATHS && process.env.HELM_CHART_PATHS.split('\n').filter(path => path != "")) || [];
+    filePathsConfig[HELM_CHART_KEY] = helmChartPaths;
+
+    //Fetch labels from each image
+    

```

Comment:
NIT: When editing test cases remove some of the new lines here.

Reply:
Done